docs/openstack/advanced-openstack-topics/openstack-cli/openstack-CLI.md

```diff
@@ -22,23 +22,29 @@ is too slow for heavy scripting use.
 To use the CLI, you must create an application credentials and set the
 appropriate environment variables.
 
-Navigate to Identity > Application Credentials
+You can download the environment file with the credentials from the OpenStack dashboard.
```

Comment:
would a link to the OpenStack Dashboard be helpful here?

Reply:
Added link to : https://stack.nerc.mghpcc.org/dashboard/identity/application_credentials/